sway-core/src/ir_generation/function.rs

```diff
@@ -200,6 +198,12 @@ fn calc_addr_as_ptr(
 }
 
 impl<'a> FnCompiler<'a> {
+    pub(super) const BACKTRACE_FN_ARG_NAME: &'static str = "__backtrace";
+    const MAX_PANIC_ERROR_CODE: u64 = 2u64.pow(8) - 1;
```

Comment:
Having the real number here makes it searchable. 
Someone will ask "what the value XXXXX means?", and it will search in the code.
Like this is harder.